paypal/express/views.py

```diff
@@ -145,8 +145,9 @@ class CancelResponseView(RedirectView):
 
     def get(self, request, *args, **kwargs):
         basket = get_object_or_404(Basket, id=kwargs['basket_id'],
-                                   status=Basket.FROZEN)
-        basket.thaw()
+                                    owner=self.request.user)
```

Comment:
@sjolicoeur Should probably filter by owner, but maybe not remove the frozen status check